src/lib/test_utils/src/setup/test_constants/mod.rs

```diff
@@ -122,25 +122,25 @@ pub fn get_canister_wasm(canister_type: KnownPrincipalType) -> Vec<u8> {
 
     match canister_type {
         KnownPrincipalType::CanisterIdConfiguration => {
-            file_path.push("configuration.wasm");
+            file_path.push("configuration.wasm.gz");
             let mut file = File::open(&file_path)
                 .unwrap_or_else(|_| panic!("Failed to open file: {}", file_path.to_str().unwrap()));
             file.read_to_end(&mut bytes).expect("Failed to read file");
         }
         KnownPrincipalType::CanisterIdDataBackup => {
-            file_path.push("data_backup.wasm");
+            file_path.push("data_backup.wasm.gz");
             let mut file = File::open(&file_path)
                 .unwrap_or_else(|_| panic!("Failed to open file: {}", file_path.to_str().unwrap()));
             file.read_to_end(&mut bytes).expect("Failed to read file");
         }
         KnownPrincipalType::CanisterIdPostCache => {
-            file_path.push("post_cache.wasm");
+            file_path.push("post_cache.wasm.gz");
             let mut file = File::open(&file_path)
                 .unwrap_or_else(|_| panic!("Failed to open file: {}", file_path.to_str().unwrap()));
             file.read_to_end(&mut bytes).expect("Failed to read file");
         }
         KnownPrincipalType::CanisterIdUserIndex => {
-            file_path.push("user_index.wasm");
+            file_path.push("user_index.wasm.gz");
```

Comment:
Do the existing ic-state-machine tests work with these changes or we are not running them ? 

Ex - `when_configuration_is_updated_in_the_configuration_canister_then_it_is_propagated_to_the_other_canisters_in_the_subnet_such_as_the_user_index_canister_and_the_post_cache_canister_and_the_individual_user_canisters` is still not [ignore]